cluster/gce/gci/configure-helper.sh

```diff
@@ -473,6 +473,7 @@ function start-kubelet {
   flags+=" --cluster-dns=${DNS_SERVER_IP}"
   flags+=" --cluster-domain=${DNS_DOMAIN}"
   flags+=" --config=/etc/kubernetes/manifests"
+  flags+=" --experimental-mounter-path=${KUBE_HOME}/bin/mounter"
```

Comment:
The `mounter`? It is installed as part of cloud init. https://github.com/vishh/kubernetes/blob/3bd69a02563a2f3bc0a9aadd0f5ea059fcd06337/cluster/gce/gci/configure.sh#L183